storage/grpc_client.go

```diff
@@ -0,0 +1,217 @@
+func (c *grpcStorageClient) ListDefaultObjectACLs(ctx context.Context, bucket string, opts ...storageOption) ([]ACLRule, error) {
+	return nil, ErrMethodNotSupported
+}
+func (c *grpcStorageClient) UpdateDefaultObjectACL(ctx context.Context, opts ...storageOption) (*ACLRule, error) {
+	return nil, ErrMethodNotSupported
+}
+
+// Bucket ACL methods.
+
+func (c *grpcStorageClient) DeleteBucketACL(ctx context.Context, bucket string, entity ACLEntity, opts ...storageOption) error {
+	return ErrMethodNotSupported
+}
+func (c *grpcStorageClient) ListBucketACLs(ctx context.Context, bucket string, opts ...storageOption) ([]ACLRule, error) {
+	return nil, ErrMethodNotSupported
+}
+func (c *grpcStorageClient) UpdateBucketACL(ctx context.Context, bucket string, entity ACLEntity, role ACLRole, opts ...storageOption) (*ACLRule, error) {
+	return nil, ErrMethodNotSupported
+}
+
+// Object ACL methods.
+
+func (c *grpcStorageClient) DeleteObjectACL(ctx context.Context, bucket, object string, entity ACLEntity, opts ...storageOption) error {
+	return ErrMethodNotSupported
+}
+func (c *grpcStorageClient) ListObjectACLs(ctx context.Context, bucket, object string, opts ...storageOption) ([]ACLRule, error) {
+	return nil, ErrMethodNotSupported
+}
+func (c *grpcStorageClient) UpdateObjectACL(ctx context.Context, bucket, object string, entity ACLEntity, role ACLRole, opts ...storageOption) (*ACLRule, error) {
+	return nil, ErrMethodNotSupported
+}
+
+// Media operations.
+
+func (c *grpcStorageClient) ComposeObject(ctx context.Context, req *composeObjectRequest, opts ...storageOption) (*ObjectAttrs, error) {
+	return nil, ErrMethodNotSupported
+}
+func (c *grpcStorageClient) RewriteObject(ctx context.Context, req *rewriteObjectRequest, opts ...storageOption) (*rewriteObjectResponse, error) {
+	return nil, ErrMethodNotSupported
+}
+
+func (c *grpcStorageClient) OpenReader(ctx context.Context, r *Reader, opts ...storageOption) error {
+	return ErrMethodNotSupported
+}
+func (c *grpcStorageClient) OpenWriter(ctx context.Context, w *Writer, opts ...storageOption) error {
+	return ErrMethodNotSupported
+}
+
+// IAM methods.
+
+func (c *grpcStorageClient) GetIamPolicy(ctx context.Context, resource string, version int32, opts ...storageOption) (*iampb.Policy, error) {
+	return nil, ErrMethodNotSupported
+}
+func (c *grpcStorageClient) SetIamPolicy(ctx context.Context, resource string, policy *iampb.Policy, opts ...storageOption) error {
+	return ErrMethodNotSupported
+}
+func (c *grpcStorageClient) TestIamPermissions(ctx context.Context, resource string, permissions []string, opts ...storageOption) ([]string, error) {
+	return nil, ErrMethodNotSupported
+}
+
+// HMAC Key methods.
+
+func (c *grpcStorageClient) GetHMACKey(ctx context.Context, desc *hmacKeyDesc, opts ...storageOption) (*HMACKey, error) {
+	return nil, ErrMethodNotSupported
+}
+func (c *grpcStorageClient) ListHMACKey(ctx context.Context, desc *hmacKeyDesc, opts ...storageOption) *HMACKeysIterator {
+	return nil
```

Comment:
Let's return an empty iterator rather than nil, I think this would make the example panic.